test/block/blockupload.test.js

```diff
@@ -612,12 +612,8 @@ describe('Block Upload', function () {
         };
 
         assert.deepStrictEqual(events.filestart[0], fileEventData);
-        assert.deepStrictEqual(events.fileprogress[0], {
-            ...fileEventData,
-            mimeType: "image/jpeg",
-            transferred: 10
-        });
-        assert.deepStrictEqual(events.fileprogress[1], {
+        assert.deepStrictEqual(events.fileprogress.length, 2);
+        assert.deepStrictEqual(events.fileprogress[events.fileprogress.length-1], {
```

Comment:
I don't quite understand this change and seems to just be missing the intermediate update part which isn't tested.

Reply:
Actually I chose to just test that everything got transferred in the end (last event), and that there were the expected number of fileprogress events (to avoid having tests containing an "or" checking for the two possible values).